nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -998,6 +998,7 @@ def __init__(
                                 f'Got: {self.activations_checkpoint_num_layers}. Setting to 1 by default.'
                             )
                         )
+                        self.activations_checkpoint_num_layers = 1  # keeping the old default
```

Comment:
this code should not execute since TransformerConfig enforces recompute_num_layers is not None, so just used the older default instead of changing it